### PR TITLE
Add 1.18.4 changelog

### PR DESCRIPTION
**Pull request by lunny**
Feel free to change the content. @go-gitea/maintainers

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,43 @@ This changelog goes through all the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
+## [1.18.4](https://github.com/go-gitea/gitea/releases/tag/1.18.4) - 2023-02-20
+
+* SECURITY
+  * Provide the ability to set password hash algorithm parameters (#22942) (#22943)
+* ENHANCEMENTS
+  * Add command to bulk set must-change-password (#22823) (#22928)
```
Comment (delvh) [inline CHANGELOG.md:11]: Isn't that `SECURITY` as well?

**Review by zeripath — APPROVED** [review]

**Review by yardenshoham — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (delvh) [inline CHANGELOG.md]:
```suggestion
  * Provide the ability to set password hash algorithm parameters (#22942) (#22943)
  * Add command to bulk set must-change-password (#22823) (#22928)
* ENHANCEMENTS
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,43 @@ This changelog goes through all the changes that have been made in each release
...
+  * escape filename when assemble URL (#22850) (#22871)
+  * Fix isAllowed of escapeStreamer (#22814) (#22837)
+  * Load issue before accessing index in merge message (#22822) (#22830)
+  * Improve trace logging for pulls and processes (#22633) (#22812)
+  * Fix restore repo bug, clarify the problem of ForeignIndex (#22776) (#22794)
+  * add default user visibility to cli command "admin user create" (#22750) (#22760)
+  * Escape path for the file list (#22741) (#22757)
+  * Fix bugs with WebAuthn preventing sign in and registration. (#22651) (#22721)
+  * Add missing close bracket in imagediff (#22710) (#22712)
+  * some refactor about code comments(#20821) (#22707)
```
Comment (wxiaoguang) [inline CHANGELOG.md]: Is this line clear?

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,43 @@ This changelog goes through all the changes that have been made in each release
...
+  * Provide the ability to set password hash algorithm parameters (#22942) (#22943)
+  * Add command to bulk set must-change-password (#22823) (#22928)
+* ENHANCEMENTS
+  * Use import of OCI structs (#22765) (#22805)
+  * Fix color of tertiary button on dark theme (#22739) (#22744)
+  * Link issue and pull requests status change in UI notifications directly to their event in the timelined view. (#22627) (#22642)
+* BUGFIXES
+  * Notify on container image create (#22806) (#22965)
+  * Fix blame view missing lines (#22826) (#22929)
+  * fix incorrect role labels for migrated issues and comments (#22914) (#22923)
```
Comment (wxiaoguang) [inline CHANGELOG.md]: Upper case for first letter? (and many below)

File: CHANGELOG.md (hunk 2 above)
Comment (lunny) [reply CHANGELOG.md]: Done.

File: CHANGELOG.md (hunk 3 above)
Comment (lunny) [reply CHANGELOG.md]: Doen.

**Comment by lunny** [comment]
LG-TM bot

**Pull request merged**